templates/resources/elasticsearch_data_stream_lifecycle.md.tmpl

```diff
@@ -0,0 +1,17 @@
+---
+subcategory: "Index"
+layout: ""
+page_title: "Elasticstack: elasticstack_elasticsearch_data_stream_lifecycle Resource"
+description: |-
+  Manages Lifecycle for Elasticsearch Data Streams
+---
+
+# Resource: elasticstack_elasticsearch_data_stream
+
+Configures the data stream lifecycle for the targeted data streams, see: https://www.elastic.co/guide/en/elasticsearch/reference/current/data-stream-apis.html
+
+## Example Usage
+
+{{ tffile "examples/resources/elasticstack_elasticsearch_data_stream_lifecycle/resource.tf" }}
+
+{{ .SchemaMarkdown | trimspace }}
```

Comment:
Can we include an [import example](https://github.com/elastic/terraform-provider-elasticstack/blob/main/templates/resources/elasticsearch_data_stream.md.tmpl#L19-L23) here as well